3.8/aql/functions-string.md

```diff
@@ -133,6 +133,18 @@ value used is 0xFFFFFFFF, and the final xor value is also 0xFFFFFFFF.
 CRC32("foobar") // "D5F5C7F"
 ```
 
+NORMALIZE_UTF8
+--------------
+
+`NORMALIZE_UTF8('some utf8 text')`
```

Comment:
I think this signature does not match the rest of the AQL functions signatures in the docs.
It should rather be something like
```
NORMALIZE_UTF8(string) → normalized
```
along with a description of the input parameter and the return value. Basically like how it is for all other AQL functions.